packages/framework/fluid-static/src/compatibilityConfiguration.ts

```diff
@@ -3,48 +3,27 @@
  * Licensed under the MIT License.
  */
 
-import {
-	CompressionAlgorithms,
-	type IContainerRuntimeOptionsInternal,
-} from "@fluidframework/container-runtime/internal";
-import { FlushMode } from "@fluidframework/runtime-definitions/internal";
+import type { IContainerRuntimeOptionsInternal } from "@fluidframework/container-runtime/internal";
 
 import type { CompatibilityMode } from "./types.js";
 
 /**
- * Specifies the configured runtime options for each {@link CompatibilityMode}..
+ * The CompatibilityMode selected determines the set of runtime options to use. In "1" mode we support
+ * full interop with true 1.x clients, while in "2" mode we only support interop with 2.x clients.
+ *
+ * @remarks In general, we can use the `compatibilityMode` property of `LoadContainerRuntimeParams` to apply
```

Comment:
```suggestion
 * @privateRemarks In general, we can use the `compatibilityMode` property of `LoadContainerRuntimeParams` to apply
```

In general, prefer remarks blocks for remarks that are useful to consumers of the API (especially when they're possibly working outside of FF) and privateRemarks for other comments (e.g. those useful to developers looking to change implementations)